<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDREAM3DMontageDialog</class>
 <widget class="QDialog" name="ImportDREAM3DMontageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import DREAM3D Montage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="6" column="0" colspan="4">
    <widget class="VSLoadHDF5DataWidget" name="loadHDF5DataWidget" native="true"/>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="cellAttrMatrixNameLE">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Cell Attribute Matrix</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="selectButton">
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number of Rows</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="tileOverlapSB">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>15</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="changeSpacingCB">
     <property name="text">
      <string>Change Spacing (X, Y, Z)</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="spacingZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="numOfColsSB">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="montageNameLE">
     <property name="text">
      <string>Untitled Montage</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Tile Overlap</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="originY">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="originX">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="spacingY">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="changeOriginCB">
     <property name="text">
      <string>Change Origin (X, Y, Z)</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="spacingX">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>Montage Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="dataFileLE">
     <property name="placeholderText">
      <string>Enter or select the data file to open</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLineEdit" name="originZ">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="numOfRowsSB">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Data Display Type</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QComboBox" name="dataDisplayTypeCB">
     <item>
      <property name="text">
       <string>Outline Only</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Side-By-Side</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Montage</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number of Columns</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Cell Attribute Matrix Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Image Data Array Name</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="imageArrayNameLE">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Image Data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VSLoadHDF5DataWidget</class>
   <extends>QWidget</extends>
   <header>VSLoadHDF5DataWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportDREAM3DMontageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportDREAM3DMontageDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
